build/config/compiler/compiler.gni

```diff
@@ -31,4 +31,7 @@ declare_args() {
 
   # Enable position independent executables (-pie).
   enable_pie = current_os == "linux"
+
+  # Remove unwind tables from the binary to save space.
+  exclude_unwind_tables = current_os != "android"
```

Comment:
nit: should we use 'include_unwind_tables' as a flag instead, to not have negatioons? All other flags seem to positive like 'enable_*', 'optimize_*'

Reply:
My guilty admission is that I copied the arguments from chromium with the aim of keeping some consistency across projects that use GN.